cli/magic-config.ts

```diff
@@ -755,28 +770,44 @@ async function processCreateOptions(options: any): Promise<void> {
           enterprise: false,
         },
       },
+      crossEncodingEnabled: answers.enableEmbeddingModelsViaSagemaker,
       embeddingsModels: [{}],
       crossEncoderModels: [{}],
     },
   };
 
+  if (
+    answers.enableEmbeddingModelsViaSagemaker &&
+    answers.enableSagemakerModels
+  ) {
+    config.rag.crossEncoderModels[0] = {
+      provider: "sagemaker",
+      name: "cross-encoder/ms-marco-MiniLM-L-12-v2",
+      default: true,
+    };
+  } else {
+    config.rag.crossEncoderModels[0] = {
+      provider: "None",
+      name: "None",
+      default: true,
+    };
+  }
```

Comment:
This configuration should not be conditional on the fact that embedding models have been enabled via SM. Always set:
```
config.rag.crossEncoderModels[0] = {
      provider: "sagemaker",
      name: "cross-encoder/ms-marco-MiniLM-L-12-v2",
      default: true,
    };
```

Reply:
We should make it clear that the cross encoding is not enabled. By setting the model as None, user would get to know that there is no cross encoder model. I will change the if condition to check for `config.rag.crossEncodingEnabled`

Reply:
Please correct as I mentioned, this configuration just reflects which are the cross-encoder models that the solution can use and is independent from the setting `config.rag.crossEncodingEnabled`. The behaviour should be the same as for the embedding models. If we disable sagemaker embedding models we are still leaving the configuration for the embbeddings.